stats/stats/src/charts/counters/yesterday_txns.rs

```diff
@@ -0,0 +1,94 @@
+use std::ops::Range;
+
+use crate::{
+    data_source::{
+        kinds::{
+            local_db::DirectPointLocalDbChartSource,
+            remote_db::{RemoteDatabaseSource, RemoteQueryBehaviour, StatementFromRange},
+        },
+        UpdateContext,
+    },
+    lines::NewTxnsStatement,
+    types::TimespanValue,
+    utils::day_start,
+    ChartProperties, MissingDatePolicy, Named, UpdateError,
+};
+use chrono::{DateTime, Days, NaiveDate, Utc};
+use entity::sea_orm_active_enums::ChartType;
+use sea_orm::FromQueryResult;
+
+pub struct YesterdayTxnsQuery;
+
+impl RemoteQueryBehaviour for YesterdayTxnsQuery {
+    type Output = TimespanValue<NaiveDate, String>;
+
+    async fn query_data(
+        cx: &UpdateContext<'_>,
+        _range: Option<Range<DateTime<Utc>>>,
+    ) -> Result<Self::Output, UpdateError> {
+        let today = cx.time.date_naive();
+        let yesterday = today
+            .checked_sub_days(Days::new(1))
+            .ok_or(UpdateError::Internal(
+                "Update time is incorrect: ~ minimum possible date".into(),
+            ))?;
+        let yesterday_range = day_start(&yesterday)..day_start(&today);
+        let query = NewTxnsStatement::get_statement(
+            Some(yesterday_range),
+            &cx.blockscout_applied_migrations,
+        );
+        let data = Self::Output::find_by_statement(query)
+            .one(cx.blockscout)
+            .await
+            .map_err(UpdateError::BlockscoutDB)?
+            // no transactions for yesterday
+            .unwrap_or(TimespanValue::with_zero_value(yesterday));
+        Ok(data)
+    }
+}
+
+pub type YesterdayTxnsRemote = RemoteDatabaseSource<YesterdayTxnsQuery>;
+
+pub struct Properties;
+
+impl Named for Properties {
+    fn name() -> String {
+        "yesterdayTxns".into()
+    }
+}
+
+impl ChartProperties for Properties {
+    type Resolution = NaiveDate;
+
+    fn chart_type() -> ChartType {
+        ChartType::Counter
+    }
+    fn missing_date_policy() -> MissingDatePolicy {
+        MissingDatePolicy::FillPrevious
```

Comment:
why do u need to specify missing_date_policy for counter? maybe leave default trait value?

Reply:
so you suggest to display zero value from midnight until the update is finished?

Reply:
because [this is what it means](https://github.com/blockscout/blockscout-rs/blob/119090cf683a7e3eb0ab1abdb868fd4add9f2121/stats/stats/src/charts/db_interaction/read.rs#L126)

Reply:
resolving bc vacation + answer given 🙈